Views/Admin/AdminProductView.cshtml

```diff
@@ -23,6 +34,18 @@
                 <div class="hs-searchbox">
                     <input type="text" class="form-control" autocomplete="off">
                 </div>
+<<<<<<< HEAD
```

Comment:
čia panašu, kad nesumergint'a gerai, <<<<<< HEAD matysis view'e, bet tikriausiai nesvarbu, nes vis vien perdarysiu šitą viewsą